website/source/docs/drivers/docker.html.md

```diff
@@ -894,7 +911,32 @@ need a higher degree of isolation between processes for security or other
 reasons, it is recommended to use full virtualization like
 [QEMU](/docs/drivers/qemu.html).
 
-## Docker for Windows Caveats
+## Caveats
+
+### Dangling Containers
+
+Nomad 0.10.2 introduces a detector and a reaper for docker dangling containers,
+containers that Nomad starts yet does not manage or track. Though rare, they
+sometimes in very loaded clusters and lead to unexpectedly running services,
```

Comment:
I feel like more than load can cause them (Nomad crashes, Nomad bugs, dockerd bugs), so maybe remove this phrase and get right to impact?

> sometimes in very loaded clusters

_clusters_ isn't really accurate here, since we binpack a cluster may be relatively idle while the few nodes that are tightly packed with allocs experience dangling containers.